packages/react/src/components/Layer/index.js

```diff
@@ -9,19 +9,28 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 import { usePrefix } from '../../internal/usePrefix';
+import { LayerContext } from './LayerContext';
```

Comment:
@joshblack I think this is why the CI is failing. Does `LayerContext` path need to be qualified further being inside the `next` folder?

Reply:
Good spot! Updated